Common/Tools/TrackTuner.h

```diff
@@ -98,11 +122,31 @@ struct TrackTuner {
   std::unique_ptr<TGraphErrors> grDcaZPullVsPtPionMC;
   std::unique_ptr<TGraphErrors> grDcaZPullVsPtPionData;
 
-  /// @brief Function to configure the TrackTuner parameters
+  /// @brief Function doing a few sanity-checks on the configurations
+  void checkConfig()
+  {
+    /// check configuration source
+    if (isConfigFromString && isConfigFromConfigurables) {
+      LOG(fatal) << " [ isConfigFromString==kTRUE and isConfigFromConfigurables==kTRUE ] Configuration done both via string and via configurables -> Only one of them can be set to kTRUE at once! Please refer to the trackTuner documentation.";
+    }
+    /// check Q/pt update
+    if ((updateCurvatureIU) && (updateCurvature)) {
+      LOG(fatal) << " [ updateCurvatureIU==kTRUE and updateCurvature==kTRUE ] -> Only one of them can be set to kTRUE at once! Please refer to the trackTuner documentation.";
+    }
+  }
+
+  /// @brief Function to configure the TrackTuner parameters with an input string
   /// @param inputString Input string with all parameter configuration. Format: <name>=<value>|<name>=<value>
   /// @return String with the values of all parameters after configurations are listed, to cross check that everything worked well
   std::string configParams(std::string inputString)
   {
+
```

Comment:
Do you really need such a banner?

Reply:
Hi @ktf , it's just a debug `LOG(info)` done once in the full execution.
I implemented it to have a reference in the log about which `configParams` function is used (with or w/o input `std::string`), since this is related to which source is used for the configuration.